content/broker/broker-transition/usage/rest/_index.en.md

```diff
@@ -0,0 +1,207 @@
+    "ReceiptHistory": null,
+    "SubReceipts": null
+}
+```
+#### Differences
+
+For Altinn 3 transitioned services the receipt is a pseudo receipt built from Altinn 3 Broker File metadata. The pseudo receipt will not contain subreceipts for the recipients.
+
+## Inbox (recipient)
+
+### Verify if users have files available
+This operation verifies whether any of the given recipients have any files available for download for the given ServiceCode / ServiceEdition combination.
+
+If files are available it returns true, otherwise it returns false.
+
+Implementation does not differ for Altinn 3 transitioned services.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/brokerservice/inbox/hasavailablefiles?serviceCode={serviceCode}&serviceEditionCode={serviceEdition}&recipients={recipient1},{recipient2} HTTP/1.1 
+accept: application/json
+ApiKey: myKey
+```
+
+### Get recipient inbox
+This operation retrieves a list of files available to the user.
+
+This operation will only return file list for transitioned services if the service is specified in the request.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/inbox/?serviceCode={serviceCode}&serviceEditionCode={serviceEdition} HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+#### Example receipt
+```JSON
+{
+    "ReceiptID": 0,
+    "ParentReceiptID": null,
+    "LastChanged": "2024-04-24T09:23:40.806",
+    "Status": "Ok",
+    "Text": "A file has been made available for download.",
+    "SendersReference": null,
+    "ServiceOwnerPartyReference": null,
+    "PartyReference": "313559017",
+    "ReceiptHistory": null,
+    "SubReceipts": null
+}
+```
+
+#### Differences
+The inbox operation will by default retrieve a list of Altinn 2 files. In order to retrieve Altinn 3 files, the service code and the service edition code parameters must be supplied.
+The receipt will not have a ReceiptId, ParentReceiptId, SendersReference or ReceiptHistory.
+
+### Get File Details inbox (recipient)
+This operation does not differ from the Get File Details for [outbox(sender)](#get-file-details-outbox-sender), except in the URI for the request.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/inbox/{fileId} HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+### Get File Receipt inbox (recipient)
+This operation does not differ from the Get File Receipt for [outbox(sender)](#get-file-receipt-outbox-sender), except in the URI for the request.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/inbox/{fileId}/receipt HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+#### Differences
+
+For Altinn 3 transitioned services the receipt is a pseudo receipt built from Altinn 3 Broker File metadata.
+
+### Download
+This operation downloads the file. It does not differ between transitioned and non-transitioned Altinn 2 services.
```

Comment:
consider using "file data" instead of file.

Reply:
done